### PR TITLE
Add a new tip for Vuejs

### PR DESCRIPTION
**Pull request by pansila**
## TL;DR;

In this tip, I will introduce an example to show you how it might conflict with other software.
## pansila

[@pansila](https://github.com/pansila)
## Extra

I'm not a native English speaker, so any grammar correction is welcome too.

### CODE REVIEW

File: _posts/en/2016-xx-xx-Vuejs-makes-a-copy-update-replace-inside-the-data-binding.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+
+title: Vuejs makes a copy-upate-replace inside the data binding
+tip-number: 60
```
Comment (zenopopovici) [inline _posts/en/2016-xx-xx-Vuejs-makes-a-copy-update-replace-inside-the-data-binding.md:5]: Update to: 69

File: _posts/en/2016-xx-xx-Vuejs-makes-a-copy-update-replace-inside-the-data-binding.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+
+title: Vuejs makes a copy-upate-replace inside the data binding
```
Comment (zenopopovici) [inline _posts/en/2016-xx-xx-Vuejs-makes-a-copy-update-replace-inside-the-data-binding.md:4]: Update to "VueJS: How VueJS makes a copy-update-replace inside the data binding."

**Review by zenopopovici — CHANGES_REQUESTED** [review]

**Comment by zenopopovici** [comment]
We disscused this and we only accept vanilla JS. Sorry.

**Comment by loverajoel** [comment]
wait, now we support categories, I like this tips.

**Comment by zenopopovici** [comment]
I was looking at an older mail with you and right now when updating a stale branch I noticed it too. Was about to re-open. Ty.

**Comment by loverajoel** [comment]
Thanks for you tip! @pansila

**Pull request merged**